_functions_jbosseap.sh

```diff
@@ -47,6 +47,8 @@ do_configure_jbosseap_datasources() {
       find_instance_file DB_SERVER_PATCH "${ETC_DIR}/${DEPLOYMENT_APPSRV_TYPE}${DEPLOYMENT_APPSRV_VERSION:0:1}" "standalone-exo-mysql.xml.patch" "${DB_SERVER_PATCH_PRODUCT_NAME}"
 
       if [ ! -f ${DEPLOYMENT_DIR}/standalone/deployments/mysql-connector*.jar ]; then
+        echo_info "Using standard mysql jdbc driver version ${DEPLOYMENT_MYSQL_DRIVER_VERSION}"
+        
         MYSQL_JAR_URL="http://repository.exoplatform.org/public/mysql/mysql-connector-java/${DEPLOYMENT_MYSQL_DRIVER_VERSION}/mysql-connector-java-${DEPLOYMENT_MYSQL_DRIVER_VERSION}.jar"
```

Comment:
this code should be moved in a dedicated function do_install_mysql_driver like for others driver types

Reply:
done